### DISCUSSION TITLE
tinydb-serialization - if I use it what impact does it have on performance?

### DISCUSSION THREAD

**Discussion by Damien455** [msiemens/tinydb#359] [Extensions]
HI, if I use tinydb-serialization and it's DateTime object ability in my database does it only have an impact on performance if I run a query on a field containing a DateTime object i.e. if a run a search on a "name" would it just be as quick if I had not used the extention?  

Many Thanks

**Comment by msiemens** [comment]
Hey @Damien455, that's really hard to answer for all possible cases. It depends on the data you're processing and how many fields have fields that contain `DateTime` objects. I guess you'd have to compare performance with and without serialization to see how big the performance impact is.

**Reply by Damien455** [reply]
thank you :)
